src/codegen/interfaces.ts

```diff
@@ -68,3 +69,21 @@ export function generateActionsNamespace(abi: ABI.Def): ts.ModuleDeclaration {
         ts.NodeFlags.Namespace
     )
 }
+
+function findParamTypeString(typeString: string, namespace: string | null, abi: ABI.Def): string {
```

Comment:
Is this method specific to interfaces? If not, I would put it in the `helper.ts` file.

Reply:
I think it is exclusive to the interfaces, since it's only used for `ActionParams` as it finds the loose types like `NameType` or `IntType`.